# Straight Line: 4m along X-axis
# Target: RMS < 0.05m

trajectory_generator:
  ros__parameters:
    waypoints: [0.0, 0.0, 1.0, 0.0, 2.0, 0.0, 3.0, 0.0, 4.0, 0.0]
    num_samples: 200
    desired_velocity: 0.20
    frame_id: 'odom'
    use_sim_time: true

trajectory_controller:
  ros__parameters:
    lookahead_distance: 0.30
    goal_tolerance: 0.05
    use_sim_time: true

trajectory_monitor:
  ros__parameters:
    csv_output_path: 'trajectory_metrics_straight'
    use_sim_time: true


